### PR TITLE
(#789) Add version normalization page

### LINKED ISSUE #789: Document version number normalization
## What New Or Updated Would You Like To See?

Add a page to the documentation explaining Version number normalization.

## Why Is It Needed?

With the upgrade to V2, version numbers are normalized. Currently we only explain this on the migration guide, but we should explain this on it's own page as it will exist beyond the initial migration.

## Additional Context?

https://github.com/chocolatey/docs/pull/764#discussion_r1268279821

### PR DESCRIPTION
**Pull request by pauby**
## Description Of Changes
Added a new version normalization page.

## Motivation and Context
The page should live separately from the upgrade guide, so it can be linked to in a better context.

## Testing

* [x] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.

## Change Types Made

* [ ] Minor documentation fix (typos etc.).
* [x] Major documentation change (refactoring, reformatting or adding documentation to existing page).
* [ ] New documentation page added.
* [ ] The change I have made should have a video added, and I have raised an issue for this.
    * Issue #

## Change Checklist

* [ ] Requires a change to menu structure (top or left hand side)/
* [ ] Menu structure has been updated

## Related Issue

Fixes #789

### CODE REVIEW

File: input/en-us/choco/features/version-number-normalization.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+Order: 20
+xref: version-normalization
+Title: Version Number Normalization
+Description: Information about the version number normalization introduced in Chocolatey CLI 2.0.0.
+---
+
+The release of Chocolatey CLI 2.0.0 brought with it new semantic version number requirements. To comply with these new requirements, Chocolatey CLI will normalize version numbers to at least three parts, and remove leading zeroes.
+
+* Version numbers that have fewer than three parts will have missing parts added, as `0` (for example, `1.2` will be normalized to `1.2.0`).
+* Version numbers that have leading zeroes in any part will have those leading zeroes removed (for example, `1.001.2` will be normalized to `1.1.2`).
```
Comment (TheCakeIsNaOH) [inline input/en-us/choco/features/version-number-normalization.md:11]: Also, four part versions get normalized to three parts, if the forth part is zero so `1.0.0.0` gets normalized to `1.0.0` but `1.0.0.1` stays the way it is.

Comment (pauby) [reply input/en-us/choco/features/version-number-normalization.md:12]: 👍 Updated.

File: input/en-us/choco/features/version-number-normalization.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+The release of Chocolatey CLI 2.0.0 brought with it new semantic version number requirements. To comply with these new requirements, Chocolatey CLI will normalize version numbers to at least three parts, and remove leading zeroes.
+
+* Version numbers that have fewer than three parts will have missing parts added, as `0` (for example, `1.2` will be normalized to `1.2.0`).
+* Version numbers that have four parts, will have the fourth part removed if it is `0` (for example, `1.0.0.0` will be normalized to `1.0.0`, but `1.0.0.1` will not).
+* Version numbers that have leading zeroes in any part will have those leading zeroes removed (for example, `1.001.2` will be normalized to `1.1.2`).
+
+This normalization is applied to the generated package when using `choco pack` under Chocolatey CLI 2.x onwards, and will also affect any displayed versions of packages from remote sources.
+
+This is a change in how Chocolatey CLI 1.x worked with version numbers.
```
Comment (vexx32) [inline input/en-us/choco/features/version-number-normalization.md:16]:
```suggestion
This is a change from how Chocolatey CLI 1.x worked with version numbers.
```

This at first glance read to me as though we're implying the normalization changed at some point during 1.x, so I think this change is a little clearer?

Comment (pauby) [reply input/en-us/choco/features/version-number-normalization.md:16]: Fixed.

**Review by corbob — APPROVED** [review]

**Review by vexx32 — APPROVED** [review]

**Pull request merged**